### DISCUSSION TITLE
Solax X3-Hybrid-10.0-d-e H3UE.... with CHINT DTSU666 @ 9600 baud rate

### DISCUSSION THREAD

**Discussion by mmayer11** [wills106/homeassistant-solax-modbus#734] [Q&A]
Hi!
I´m trying to get my Solax X3 Hybrid hooked up to Will´s Solax HA integration.
According to petrs99 discussion my model should talk Modbus TCP (Ethernet) right out it´s LAN or COM port.
https://github.com/wills106/homeassistant-solax-modbus/discussions/303#discussioncomment-4750980
![grafik](https://github.com/wills106/homeassistant-solax-modbus/assets/149958510/a25ba3f7-66b6-4b4e-8172-4fd0bfb44b0e)

Unfortunately in my case in HA no devices or entities are getting discovered of my Solax inverter after I configure the IP adress and the other settings of the inverter. 
I am guessing that this might be due to the CHINT power meter that is operating with max 9600 baud (the waveshare modbus tcp converters are set to 19200)
Can somebody pls. help me out on this?
Thx!

**Comment by wills106** [comment]
Are you trying to connect a Waveshare to your Inverter? Or directly connecting Ethernet from the Inverter to your switch / router?

**Reply by mmayer11** [reply]
It´s a direct Ethernet connection from the inverter to a switch and I´m able to ping the inverter with it´s (fixed) IP address.
No Waveshare in the game...
